src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Commands/LogMessagesPage.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Specialized;
+using Microsoft.CommandPalette.Extensions;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.UI.ViewModels.Commands;
+
+public partial class LogMessagesPage : ListPage
+{
+    private readonly List<IListItem> _listItems = new();
+
+    public LogMessagesPage()
+    {
+        Name = Properties.Resources.builtin_log_name;
+        Title = Properties.Resources.builtin_log_page_name;
+        Icon = new IconInfo("\uE8FD");
```

Comment:
Comment that this is BulletedList icon. I find those helpful elsewhere.
```suggestion
        Icon = new IconInfo("\uE8FD"); // BulletedList icon
```